### PR TITLE
Add documentation for options in usage section

### PR DESCRIPTION
**Pull request by jsejcksn**

### CODE REVIEW

**Comment by jsejcksn** [comment]
I was confused about the difference between the `largest` and `units` options for a while, but I finally figured it out and decided to write documentation for each of the options to  (hopefully) help guide others who might desire more clarity or verbosity. I think I understand what each one does and described them correctly, but if I got any of them wrong, then let me know what needs to change. I have found a lot of value in this library! Thanks for making it available for everyone.

**Comment by jsejcksn** [comment]
@EvanHahn How does everything look?

File: README.md [hunk 1]
```diff
@@ -73,7 +121,11 @@ humanizeDuration(400, {  // '1 year, 1 month, 5 days'
     d: 1
   }
 })
+```
 
+**Combined example**
```
Comment (EvanHahn) [inline README.md]: Could you add a few more options in this combined example?

File: README.md [hunk 2]
```diff
@@ -191,6 +243,7 @@ Lovingly made by [Evan Hahn](http://evanhahn.com/) with help from:
 * [Toni Helminen](https://github.com/tonihelminen) for Finnish support
 * [Vidmantas Drasutis](https://github.com/Drasius2) for Lithuanian support
 * [Manh Tuan](https://github.com/J2TeaM) for Vietnamese support
+* [Jesse Jackson](https://github.com/jsejcksn) for documentation support
```
Comment (EvanHahn) [inline README.md]: Could you change this to `for documentation help`?

File: README.md [hunk 3]
```diff
@@ -38,32 +38,80 @@ humanizeDuration(2015)      // '2.25 seconds'
 humanizeDuration(97320000)  // '1 day, 3 hours, 2 minutes'
 ```
 
+### Options
+
 You can change the settings by passing options as the second argument:
 
+**language**
+
+Language for unit display (accepts an [ISO 639-1 code](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) from one of the [supported languages](#supported-languages))
```
Comment (EvanHahn) [inline README.md]: Could you make these descriptions end with periods?

**Comment by EvanHahn** [comment]
👍 other than my comments. Sorry it took so long to get back to you—things have been busy!

**Comment by jsejcksn** [comment]
@EvanHahn Will do tonight or tomorrow.

**Comment by jsejcksn** [comment]
@EvanHahn Ready to go!

**Comment by EvanHahn** [comment]
This is released in Humanize Duration 3.8.0! Thanks for your contribution.

**Pull request merged**